compiler/ExtractBase.ml

```diff
@@ -2053,4 +2055,4 @@ let ctx_add_fun_decl (def : fun_decl) (ctx : extraction_ctx) : extraction_ctx =
 let ctx_compute_type_decl_name (ctx : extraction_ctx) (def : type_decl) : string
     =
   let llbc_name = rename_llbc_name def.item_meta def.llbc_name in
```

Comment:
I think this should be moved to `ctx_compute_type_decl_name_aux`, otherwise you will forget to update the LLBC name in other places